### ISSUE TITLE
Show rating details on hover

### ISSUE THREAD

**Issue by johan456789** [rubenmv/extension-goodreads-ratings-for-amazon#8]
Would be great if it can show rating details by hovering on the stars like amazon does.

![image](https://user-images.githubusercontent.com/14802181/89039786-3dd19080-d375-11ea-8b0d-52ced835132f.png)

Something like this. Rating details is available at goodreads.

![image](https://user-images.githubusercontent.com/14802181/89039976-8be69400-d375-11ea-81b7-700ac8ef5cd4.png)

**Comment by makethingsbetter** [comment]
no that's useless info

+may slow ext down

this is what helpful useful info is https://chrome.google.com/webstore/detail/reviewmetacom-review-anal/fjifglfkcaipnmhngbigdebkoikioend

**Comment by berezovskyi** [comment]
@makethingsbetter your suggested extension is great, I will take a look at it but you understand this ticket is for book reviews where people think that Goodreads users have on avaerage a more, let's say, "refined" taste in literature, while your extension tries to prevent fraudlent reviews if I understood it correctly.

**Comment by makethingsbetter** [comment]
dont assume credit where there is none, is and has been a very popular extension/site + best of it's kind, whoever made it is top notch

**Comment by berezovskyi** [comment]
Yes, I just tried and indeed the extenstion you recommend is nice. However, I stand by my assessment that its use when it comes to books is quite limited. Example: https://reviewmeta.com/amazon-de/1473637465 what does "brand monogamy" even supposed to mean? :)
